test/language/statements/try/early-catch-duplicates.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-try-statement-static-semantics-early-errors
+es6id: 13.15.1
+description: >
+    It is a Syntax Error if BoundNames of CatchParameter contains any duplicate
+    elements.
+negative: SyntaxError
+---*/
+
+$ERROR('This code should not be executed.');
```

Comment:
Some [other tests](https://github.com/tc39/test262/pull/566/files) use the `throw new Test262Error()`. Both cases are correct, but we should define one way to do it for consistency.